### ISSUE TITLE
Modify the information in an existing ROR record: Thornbury Hospital https://ror.org/008t9c882

### ISSUE THREAD

**Issue by zambujo** [ror-community/ror-updates#1924]
Summary of request: Modify the information in an existing ROR record

Name of organization: Thornbury Hospital
ROR ID: https://ror.org/008t9c882
Which part of the record needs to be changed? Organization website
Description of change: Replace https://www.bmihealthcare.co.uk/hospitals/bmi-thornbury-hospital with https://www.circlehealthgroup.co.uk/hospitals/thornbury-hospital

Update: replace.links==https://www.circlehealthgroup.co.uk/hospitals/thornbury-hospital;$

**Comment by adambuttrick** [comment]
This record is ambiguous. Address is listed as in Bristol, for which the [NHS lists a hospital by this name](https://www.nhs.uk/services/hospital/thornbury-hospital-henderson-ward/X97519). Link provided is for another hospital in Sheffield. Affiliation usage exists for both locations (Bristol - https://api.openalex.org/W2426288554, Sheffield - https://doi.org/10.1080/14992027.2020.1855370). I think for now, we should just remove this link and see if we can puzzle out from GRID usage to what this refers.

**Comment by mariagould** [comment]
I think I've figured out what's going on here. We actually have two Thornbury Hospital records in ROR, for two distinct entities:

https://ror.org/04xewk010 (referenced in this request) is for Thornbury Hospital in Bristol, which appears to have been shut down as of 2018 per https://www.bristolpost.co.uk/news/bristol-news/thornbury-resident-says-hospital-been-6100818. 

https://ror.org/008t9c882 is the record for Thornbury Hospital in Sheffield. The link in that record (https://www.bmihealthcare.co.uk/hospitals/bmi-thornbury-hospital) redirects to https://www.circlehealthgroup.co.uk/hospitals/thornbury-hospital, which was indicated by the requestor. 

Bottom line: the requested change should be made, but it should be applied to https://ror.org/008t9c882 instead of https://ror.org/04xewk010. I will update the metadata in this issue accordingly. And I think we should deactivate the record for the Bristol location (I'll create a separate issue for this).

**Comment by adambuttrick** [comment]
Record updated in release v1.18.
